tests/pytests/unit/modules/test_slack.py

```diff
@@ -0,0 +1,84 @@
+"""
+Tests for salt.modules.slack module
+"""
+
+import pytest
+import salt.modules.slack_notify as slack
+from salt.ext.six.moves.urllib.parse import urlencode as _urlencode
```

Comment:
```suggestion
from urllib.parse import urlencode as _urlencode
```

Maybe even better,
```suggestion
import urllib.parse
```
And use the fully qualified module name and function

Reply:
Great! Thanks for the info, I'll fix this and get back to you.

Reply:
Done